### DISCUSSION TITLE
My card was charged twice after change the payment date

### DISCUSSION THREAD

**Discussion by hieuhuynh93** [community/community#109952] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hi,
Previously, I requested to change the payment date and it was approved. I also paid for the difference of ~$5. However, my credit card was charged $10 for Copilot at mid night from Github. I also received the invoice via email. 
I submitted the ticket but no one replied in 2 days (Ticket ID #2615164).
I need to resolve this issue as soon as possible.
I really don't feel safe paying here.

![P7Gbxhls-2024-02-25 at 02 53 12@2x](https://github.com/community/community/assets/10773568/1cf6c6d7-34be-4842-8261-b025563627f5)

**Comment by davecheney** [comment, marked as answer]
@hieuhuynh93 i'm sorry that you experienced this. We cannot resolve billing issues via this forum, please contact support https://support.github.com/contact

**Reply by hieuhuynh93** [reply]
I submitted the ticket but no one replied in 2 days (Ticket ID #2615164).
